lib/sober_swag/reporting/output/partitioned.rb

```diff
@@ -0,0 +1,63 @@
+module SoberSwag
+  module Reporting
+    module Output
+      ##
+      # Partition output into one of two possible cases.
+      class Partitioned < Base
+        ##
+        # @param partition [#call] block that returns true or false for the input type
```

Comment:
nit: What does true or false actually mean under this circumstance?

Reply:
0ef234a